src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  today: Date,
+};
+
+export class Clock extends React.Component<Props> {
+  state: State = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentWillUnmount(): void {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {this.props.name}
+        </strong>
+
+        {' time is '}
+
+        <span className="Clock__time">
+          {this.state.today.toUTCString().slice(-12, -4)}
```

Comment:
You make similar calculations twice, so you should make a separate helper function for this.